iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Authentication;
+using Microsoft.Azure.Devices.Extensions;
+using Microsoft.Azure.Devices.Generated;
+using Microsoft.Rest;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// /// The Digital Twins Service Client contains methods to retrieve and update digital twin information, and invoke commands on a digital twin device.
+    /// </summary>
+    public class DigitalTwinClient : IDisposable
+    {
+        private readonly IotHubGatewayServiceAPIs _client;
+        private readonly DigitalTwin _protocolLayer;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DigitalTwinClient"/> class.</summary>
+        /// <param name="connectionString">Your IoT hub's connection string.</param>
```

Comment:
Just "IoT hub's connection string" I don't think we should use "Your" in the summary.